README.md

```diff
@@ -13,11 +13,17 @@
+ 9  | Thu, Jan 03, 2019 14:00 UTC       | [agenda](https://github.com/ethresearch/eth2.0-pm/issues/21) \| [notes](eth2.0-implementers-calls/call_009.md) \| [reddit](https://www.reddit.com/r/ethereum/comments/ac1smo/livestream_eth20_implementers_call_9_201913_2pm/)  | [video](https://www.youtube.com/watch?v=6trA-5rjZUQ) |
  10 | Thu, Jan 17, 2019 14:00 UTC      | [agenda](https://github.com/ethresearch/eth2.0-pm/issues/23) \| [notes](eth2.0-implementers-calls/call_010.md) \| [reddit](https://www.reddit.com/r/ethereum/comments/agtomc/live_eth20_implementers_call_10_20190116_1400_utc/)  | [video](https://www.youtube.com/watch?v=KZ9fms_PrQU) |
  11 | Thu, Jan 31, 2019 14:00 UTC      | [agenda](https://github.com/ethresearch/eth2.0-pm/issues/27) \| [notes](eth2.0-implementers-calls/call_011.md) \| [reddit]()  | [video](https://www.youtube.com/watch?v=wS3sOB_hfgk) |
  12 | Thu, Feb 14, 2019 14:00 UTC      | [agenda](https://github.com/ethresearch/eth2.0-pm/issues/29) \| [notes](eth2.0-implementers-calls/call_012.md) \| [reddit](https://www.reddit.com/r/ethereum/comments/aqe147/eth20_call_12_stream/)  | [video](https://www.youtube.com/watch?v=p1qHM2B8cGc) |
  13 | Thu, Feb 28, 2019 14:00 UTC      | [agenda](https://github.com/ethresearch/eth2.0-pm/issues/31) \| [notes](eth2.0-implementers-calls/call_013.md) \| [reddit]()  | [video](https://www.youtube.com/watch?v=0ZWG8hMbxes) |
  14 | Thu, Mar 14, 2019 14:00 UTC      | [agenda](https://github.com/ethresearch/eth2.0-pm/issues/33) \| [notes](eth2.0-implementers-calls/call_014.md) \| [reddit](https://www.reddit.com/r/ethereum/comments/b0ud27/live_eth20_implementers_call_14_201903214_starts/)  | [video](https://www.youtube.com/watch?v=zeceWlmxseY) |
- 15 | Thu, Mar 28, 2019 14:00 UTC      | [agenda](https://github.com/ethresearch/eth2.0-pm/issues/35) \| [notes](eth2.0-implementers-calls/call_015.md) \| [reddit]()  | [video](https://www.youtube.com/watch?v=bC4v_a-gcrs) |
- 16 | Thu, Apr 18, 2019 14:00 UTC      | [agenda](https://github.com/ethereum/eth2.0-pm/issues/37) \| [notes](https://github.com/ethereum/eth2.0-pm/blob/master/eth2.0-implementers-calls/call_016.md) \| [reddit]()  | [video](https://www.youtube.com/watch?v=eN_O8bSaS5Q) |
+ 15 | Thu, Mar 28, 2019 14:00 UTC      | [agenda](https://github.com/ethresearch/eth2.0-pm/issues/35) \| [notes](eth2.0-implementers-calls/call_015.md) \| [video](https://www.youtube.com/watch?v=bC4v_a-gcrs) |
+ 16 | Thu, Apr 18, 2019 14:00 UTC      | [agenda](https://github.com/ethereum/eth2.0-pm/issues/37) \| [notes](https://github.com/ethereum/eth2.0-pm/blob/master/eth2.0-implementers-calls/call_016.md) \| [video](https://www.youtube.com/watch?v=eN_O8bSaS5Q) |
```

Comment:
Let's keep the final column for "Video" and if there is no reddit link available, put "No Reddit"

Reply:
Updated to reflect No Reddit. There was a typo in Meeting 16 that resulted in the video not appearing the in the last column.